apps/emqx_plugins/src/emqx_plugins.erl

```diff
@@ -35,43 +44,61 @@
     delete_package/1
 ]).
 
+%% Plugin runtime management
 -export([
     ensure_started/0,
     ensure_started/1,
     ensure_stopped/0,
     ensure_stopped/1,
     restart/1,
-    list/0,
-    describe/1,
-    parse_name_vsn/1
+    list/0
+]).
+
+%% Plugin config APIs
+-export([
+    get_plugin_config/1,
```

Comment:
the module name is emqx_plugins, so I think it would make sense to name it just get_config.
the internal (non-api) functions can be renamed in case of name clash

Reply:
it's like emqx_logger:set_log_level, should have been named set_level.